src/features/presence/PresenceCollaborator.js

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import isEmpty from 'lodash/isEmpty';
+import PropTypes from 'prop-types';
+import { FormattedMessage, injectIntl } from 'react-intl';
+
+import Link from '../../components/link/LinkBase';
+import messages from './messages';
+import PresenceAvatar from './PresenceAvatar';
+import { determineInteractionMessage } from './utils/presenceUtils';
+
+import './PresenceCollaborator.scss';
+
+export const renderTimestampMessage = (interactedAt, interactionType, intl) => {
+    const lastActionMessage = determineInteractionMessage(interactionType, interactedAt);
+    const timeAgo = intl.formatRelativeTime
+        ? intl.formatRelativeTime(interactedAt - Date.now())
+        : intl.formatRelative(interactedAt);
+
+    if (lastActionMessage) {
+        return (
+            <FormattedMessage
+                {...lastActionMessage}
+                values={{
+                    timeAgo,
+                }}
+            />
+        );
+    }
+    return null;
+};
+
+const PresenceCollaborator = ({ collaborator, intl }) => {
+    const { avatarUrl, id, interactedAt, interactionType, isActive, name, profileUrl } = collaborator;
+
+    return (
+        <div key={id} className="bdl-PresenceCollaborator" role="listitem">
```

Comment:
Can we move `role="listitem"` up to where this component is used? It may not always be used in the context of a list.

Reply:
We can also remove `key={id}`, since this is just returning a single node.